pytest_github/plugin.py

```diff
@@ -226,7 +226,7 @@ def __init__(self, username, password, completed_labels=GITHUB_COMPLETED_LABELS)
         # Process parameters
         self.username = username
         self.password = password
```

Comment:
The `completed_labels` should be set to the argument passed into the method, not the global variable